tests/keras_tests/feature_networks_tests/test_features_runner.py

```diff
@@ -537,6 +538,16 @@ def test_linear_collapsing(self):
         SixConv2DCollapsingTest(self).run_test()
         Op2DAddConstCollapsingTest(self).run_test()
 
+    def test_const_quantization(self):
```

Comment:
maybe add tests for mixed precision and gptq for a model with const quantization?

In general, since we are not updating the default TPC to v2, I think that we need to think of a way to test more MCT features with const quantization enabled, otherwise we won't have a way to identify if something is broken due to the added const quantization (maybe change the tests TPCs the use "latest" to use "v2". or something else if you have a better idea)